api/v1/helper.go

```diff
@@ -29,6 +29,13 @@ const (
 	POOLCONFIGFINALIZERNAME = "poolconfig.finalizers.sriovnetwork.openshift.io"
 	ESwithModeLegacy        = "legacy"
 	ESwithModeSwitchDev     = "switchdev"
+
+	Enable    = "enable"
```

Comment:
nit: can we prefix those as well ? 

(sorry for not adding this comment earlier)

SriovCniStateEnable
SriovCniStateDisable
SriovCniStateAuto
SriovCniSpoofCheckOff
SriovCniSpoofCheckOn
IpamEmpty

Reply:
and SriovCniIpamEmpty

Reply:
Done